logical/plugin/pb/translation.go

```diff
@@ -130,27 +130,19 @@ func ProtoLeaseOptionsToLogicalLeaseOptions(l *LeaseOptions) (logical.LeaseOptio
 		return logical.LeaseOptions{}, nil
 	}
 
-	t, err := ptypes.Timestamp(l.IssueTime)
 	return logical.LeaseOptions{
 		TTL:       time.Duration(l.TTL),
+		MaxTTL:    time.Duration(l.MaxTTL),
 		Renewable: l.Renewable,
-		Increment: time.Duration(l.Increment),
-		IssueTime: t,
-	}, err
+	}, nil
 }
 
 func LogicalLeaseOptionsToProtoLeaseOptions(l logical.LeaseOptions) (*LeaseOptions, error) {
-	t, err := ptypes.TimestampProto(l.IssueTime)
-	if err != nil {
-		return nil, err
-	}
-
 	return &LeaseOptions{
 		TTL:       int64(l.TTL),
+		MaxTTL:    int64(l.MaxTTL),
```

Comment:
We could just return l as-is or remove this function?

Reply:
I think we need to do this because the proto has this defined as in int64, so we need to translate between the proto and core.

Reply:
You might have missed this comment :-)